packages/frontend/src/components/Routing.js

```diff
@@ -329,7 +330,7 @@ class Routing extends Component {
                             <GuestLandingRoute
                                 exact
                                 path='/'
-                                render={(props) => isInactiveAccount ? <ActivateAccountWithRouter {...props}/> : <Wallet tab={tab} setTab={setTab} {...props} />}
+                                render={(props) => isInactiveAccount ? <ActivateAccountWithRouter {...props}/> : <Wallet tab={tab} setTab={setTab} history={this.props.history} {...props} />}
```

Comment:
Won't be needed anymore:) https://github.com/near/near-wallet/pull/2205/files#r758857643